snmp/datadog_checks/snmp/data/default_profiles/_dell-rac.yaml

```
@@ -78,6 +78,37 @@ metrics:
           OID: 1.3.6.1.4.1.674.10892.5.5.1.20.130.4.1.2
           name: physicalDiskName
         tag: disk_name
+  - MIB: IDRAC-MIB-SMIv2
+    table:
+      OID: 1.3.6.1.4.1.674.10892.5.5.1.20.130.4
+      name: physicalDiskTable
+    symbols:
+      - name: physicalDisk
+        constant_value_one: true
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.5.1.20.130.4.1.4
+          name: physicalDiskState
+        tag: disk_state
+        mapping:
+          1: unknown
+          2: ready
+          3: online
+          4: foreign
+          5: offline
+          6: blocked
+          7: failed
+          8: nonraid
+          9: removed
+          10: readonly
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.5.1.20.130.4.1.2
+          name: physicalDiskName
+        tag: disk_name
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.5.1.20.130.4.1.1 # 3 & 14
```

```suggestion
          OID: 1.3.6.1.4.1.674.10892.5.5.1.20.130.4.1.1
```

Shall we remove this comment?

---

Fair enough, forgot about it 😅